jib-maven-plugin/src/test/resources/maven/projects/simple/src/main/java/com/test/HelloWorld.java

```diff
@@ -34,37 +35,48 @@ public static void main(String[] args) throws IOException, URISyntaxException {
     String greeting = Greeting.getGreeting();
 
     // Gets the contents of the resource file 'world'.
-    ClassLoader classLoader = HelloWorld.class.getClassLoader();
-    Path worldFile = Paths.get(classLoader.getResource("world").toURI());
-    String world = new String(Files.readAllBytes(worldFile), StandardCharsets.UTF_8);
+    ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
As noted elsewhere, the context classloader is not recommended for use unless absolutely necessary.  It shouldn't be necessary here.